package.json

```diff
@@ -140,8 +140,8 @@
 		"prepack": "rm -rf lib && tsc -b && oclif manifest && oclif readme && oclif lock && npm shrinkwrap",
 		"postpack": "rm -f oclif.manifest.json oclif.lock npm-shrinkwrap.json",
 		"lint-typescript": "eslint ./src --ext .ts --max-warnings 0",
-		"test": "./gradlew test jacocoTestCoverageVerification && nyc mocha --timeout 10000 --retries 5 \"./test/**/*.test.ts\"",
-		"test-quiet": "cross-env SFGE_LOGGING=false ./gradlew test jacocoTestCoverageVerification && nyc mocha --timeout 10000 --retries 5 \"./test/**/*.test.ts\"",
+		"test": "./gradlew test jacocoTestCoverageVerification && nyc mocha --timeout 60000 --retries 5 \"./test/**/*.test.ts\"",
```

Comment:
why are we making this change?

Reply:
You'll notice that test-typescript already has a --timeout of 60000 ... so I was updating the others to reflect that change in all the commands. Basically on a given users machine a few tests go over 10 seconds... on my machine 3 tests consistently take 12 to 13 seconds. In the past, Josh updated test-typescript (which is the command he uses locally) to 60 seconds. I personally use yarn test locally to just run all the tests for all subprojects. So increasing this timeout to 60 seconds is the safest thing to do... it allows us to run things locally without tests dying but gives a sanity check for a run away test to stop and fail in our build system.